MyTestGen.sublime-snippet

```diff
@@ -66,14 +66,38 @@ void strmx(int mxlen, int n) {
 } 
 
 void tree(int n) {
-	int exc = rand(1,n);
-	for (int i = 1; i <= n; ++i)
-	{
-		if(i==exc) continue;
-		int first = rand(1,n);
-		while(first==i) first = rand(1,n);
-		cout<<first<<" "<<i<<endl;
-	}
+	int prufer[n-2];
+    rep(i,0,n){
```

Comment:
```suggestion
    rep(i,0,n-2){
```

Reply:
@mrsac7 I just discovered a minor error in your code. Even though the error is small, it produces very unexpected output on my PC.